### DISCUSSION TITLE
The bug of brats_segmentation_3d

### DISCUSSION THREAD

**Discussion by Achillesy** [Project-MONAI/tutorials#1140] [General]
The EnsureTyped function is missing in val_org_transforms, which may cause unknown errors in lower versions of Monai 0.9.0

**Comment by KumoLiu** [comment]
Hi @Achillesy, we recommend using metatensor from version 0.9.1, which is why `EnsureTyped` was removed. If you are using an earlier version, you can add `EnsureTyped` or change the tutorial tag to 0.9.0 to see the earlier versions of the tutorial.

**Reply by Achillesy** [reply]
1.  Both train_transform and val_transform use EnsureTyped
2.  In the same hardware environment, Geforce RTX 2080 Ti 12G, if monai > 0.9.0, the init_filter must be reduced so as not to cause GPU overflow. Your answer makes me believe that metatensor consumes more video memory.
